style.css

```diff
@@ -0,0 +1,336 @@
+height: 120px;
+left: 165px;
+top: 64px;
+}
+.left-top div{
+  position: absolute;
+  top: 120px;
+  left: 160px;
+  width: 445px;
+  height: 168px;
+  gap:8px;
+}
+.left-bottom  div img{
+  width: 16px;
+  height: 14px;
+  position: relative;
+  left: 0px;
+  bottom: -20px;
+}
+.left-bottom h3 {
+  margin-top: 50px ;
+  width: 101px;
+height: 28px;
+font-family: 'Inter';
+font-style: normal;
+font-weight: 700;
+font-size: 20px;
+line-height: 28px;
+font-feature-settings: 'salt' on, 'liga' off;
+color: #18191F;
+}
+.left-bottom div div {
+  padding-left: 25px;
+}
+.left-bottom p{
+  width: 147px;
+  height: 24px;
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  color: #969BAB;
+}
+.right-top div div {
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 32px;
+  font-feature-settings: 'salt' on, 'liga' off;
+  color: #18191F;
+width: 349px;
+height: 128px;
+margin-top:10px;
+}
+.right-top div div p{
+  width: 300px;
+height: 130px;
+font-family: 'Inter';
+font-style: normal;
+font-weight: 400;
+font-size: 18px;
+line-height: 32px;
+font-feature-settings: 'salt' on, 'liga' off;
+color: #18191F;
+margin-top: 20px;
+}
+.right-top div div h4{
+  width: 114px;
+height: 28px;
+margin-top: 30px;
+font-family: 'Inter';
+font-style: normal;
+font-weight: 700;
+font-size: 18px;
+line-height: 28px;
+font-feature-settings: 'salt' on, 'liga' off;
+color: #18191F;
+}
+.right-top div div h6{
+  width: 82px;
+height: 24px;
+font-family: 'Inter';
+font-style: normal;
+font-weight: 500;
+font-size: 14px;
+line-height: 24px;
+color: #969BAB;
+}
+.right-top div img{
+  position: absolute;
+  left: 7%;
+  right: 10%;
+  top: 32%;
+  height: 14px;
+  width: 15.929831504821777px;
+}
+.right-top div div {
+  padding: 35px 32px 32px 24px ;
+}
+.right-top img{
+  position: absolute;
+  height: 40px;
+  width: 130px;
+  left: 32px;
+  top: 32px;
+}
+.right-bottom div div{
+  padding-left: 20px;
+  width: 286px;
+  height: 140px;
+}
+.right-bottom div img{
+position: absolute;
+left:8%;
+top: 37%;
+right: 0.44%;
+height: 14px;
+width: 15.929831504821777px;
+}
+.right-bottom img{
+  width: 134px;
+height: 40px;
+margin-bottom: 20px;
+}
+.right-bottom div div  p{
+  width: 254px;
+  height: 64px;
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 29px;
+  font-feature-settings: 'salt' on, 'liga' off;
+  color: #18191F;
+}
+.right-bottom h5{
+  width: 131px;
+  height: 28px;
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 28px;
+  font-feature-settings: 'salt' on, 'liga' off;
+  color: #18191F;
+  margin: 30px 0 0px 0;
+}
+.right-bottom div div h6{
+  width: 179px;
+height: 24px;
+font-family: 'Inter';
+font-style: normal;
+font-weight: 500;
+font-size: 14px;
+line-height: 24px;
+font-feature-settings: 'salt' on, 'liga' off;
+color: #969BAB;
+}
```

Comment:
- ## Extract the styles for each section into a separate file
- ## Remove `font-feature-settings`
- ## Same thing here. no use of absolute values
> ## Use class names; meaningful class names

Reply:
- Removed font-feature-settings
- Extracted and commented inside css  the styles for each section into a separate file
- bro i couldnt adjust the last section , i have just commented the whole section because i couldnt figure out